apps/web/pages/v2/settings/security/password.tsx

```diff
@@ -45,46 +65,17 @@ const PasswordView = () => {
           </p>
         </div>
       ) : (
-        <Form
-          form={formMethods}
-          handleSubmit={async (values) => {
-            const { oldPassword, newPassword } = values;
-            mutation.mutate({ oldPassword, newPassword });
-          }}>
+        <Form<ChangePasswordFormValues> form={formMethods} handleSubmit={handleSubmit}>
           <div className="max-w-[38rem] sm:flex sm:space-x-4">
             <div className="flex-grow">
-              <Controller
-                name="oldPassword"
-                control={formMethods.control}
-                render={({ field: { value } }) => (
-                  <TextField
-                    name="oldPassword"
-                    label={t("old_password")}
-                    value={value}
-                    type="password"
-                    onChange={(e) => {
-                      formMethods.setValue("oldPassword", e?.target.value);
-                    }}
-                  />
-                )}
-              />
```

Comment:
Why are we getting rid of `Controller` here but not on other screens?

Reply:
Primarily because this one is broken, clicking the "Update" button to change the password did not do anything at all. Changing _all_ the text fields to be uncontrolled is a massive refactor.